### PR TITLE
pitr: support s3 object lock

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -214,4 +214,4 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
 
 ## 存储服务其他功能支持
 
-TiDB 备份恢复功能从 v6.3.0 支持 AWS S3 Object Lock 功能。你可以在 AWS 中开启 [S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能来防止备份数据写入后被修改或者删除。
+Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能可以帮助用户设置数据留存期，防止备份数据在一段时间内被意外或故意删除，增强了数据的安全性和完整性。自 v6.3.0 版本开始，BR 的快照备份恢复功能支持了 Amazon S3 的对象锁定功能，为数据提供了额外的安全保障。从 v8.0.0 版本起，PITR 日志备份也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性。BR/PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，用户无需进行任何额外的变更。
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:217]:
```suggestion
Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，用户无需进行任何额外的变更。
```

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -214,4 +214,4 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
 
 ## 存储服务其他功能支持
 
-TiDB 备份恢复功能从 v6.3.0 支持 AWS S3 Object Lock 功能。你可以在 AWS 中开启 [S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能来防止备份数据写入后被修改或者删除。
+Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，用户无需进行任何额外的变更。
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:217]:
```suggestion
Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。

BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，用户无需进行任何额外的变更。
```

File: br/backup-and-restore-storages.md [hunk 3]
```diff
@@ -214,4 +214,6 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
 
 ## 存储服务其他功能支持
 
-TiDB 备份恢复功能从 v6.3.0 支持 AWS S3 Object Lock 功能。你可以在 AWS 中开启 [S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能来防止备份数据写入后被修改或者删除。
+Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。
+
+BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，用户无需进行任何额外的变更。
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:219]:
```suggestion
BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，你无需进行任何额外的变更。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: br/backup-and-restore-storages.md [hunk 4]
```diff
@@ -214,4 +214,6 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
 
 ## 存储服务其他功能支持
 
-TiDB 备份恢复功能从 v6.3.0 支持 AWS S3 Object Lock 功能。你可以在 AWS 中开启 [S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能来防止备份数据写入后被修改或者删除。
+Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。
+
+BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，你无需进行任何额外的变更。
```
Comment (BornChanger) [inline br/backup-and-restore-storages.md:219]:
```suggestion
BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，你无需进行任何额外的变更。

> **警告：**
>
> 如果在快照备份执行过程中或者 PITR 日志备份过程中，才开启了对象锁定功能，快照备份或者日志备份可能会失败。需要重新开启快照备份或者是 PITR 日志备份任务来完成备份。
>
```

File: br/backup-and-restore-storages.md [hunk 5]
```diff
@@ -214,4 +214,11 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
...
+
+> **警告：**
+>
+> 如果在快照备份执行过程中或者 PITR 日志备份过程中，才开启了对象锁定功能，快照备份或者日志备份可能会失败。需要重新启动快照备份或者是 PITR 日志备份任务来继续完成备份。
+>
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:224]:
```suggestion
> 如果在快照备份执行过程中或者 PITR 日志备份过程中，才开启了对象锁定功能，快照备份或者日志备份可能会失败。需要重新启动快照备份或者是 PITR 日志备份任务来继续完成备份。
```

File: br/backup-and-restore-storages.md [hunk 6]
```diff
@@ -214,4 +214,10 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
 
 ## 存储服务其他功能支持
 
-TiDB 备份恢复功能从 v6.3.0 支持 AWS S3 Object Lock 功能。你可以在 AWS 中开启 [S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能来防止备份数据写入后被修改或者删除。
+Amazon [S3 对象锁定](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。
```
Comment (qiancai) [inline br/backup-and-restore-storages.md:217]:
```suggestion
Amazon [S3 对象锁定](https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/userguide/object-lock.html) 功能支持用户通过设置数据留存期，有效防止备份数据在指定时间内被意外或故意删除，提升了数据的安全性和完整性。从 v6.3.0 起，BR 为快照备份引入了对 Amazon S3 对象锁定功能的支持，为全量备份增加了额外的安全性保障。从 v8.0.0 起，PITR 也引入了对 Amazon S3 对象锁定功能的支持，无论是全量备份还是日志数据备份，都可以通过对象锁定功能提供更可靠的数据保护，进一步加强了数据备份和恢复的安全性，并满足了监管方面的需求。
```

File: br/backup-and-restore-storages.md [hunk 7]
```diff
@@ -214,4 +214,10 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
...
+
+BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，你无需进行任何额外的变更。
```
Comment (qiancai) [inline br/backup-and-restore-storages.md:219]:
```suggestion
BR 和 PITR 将自动检测 Amazon S3 对象锁定功能的开启或关闭状态，你无需进行任何额外的操作。
```

File: br/backup-and-restore-storages.md [hunk 8]
```diff
@@ -214,4 +214,10 @@ TiDB 备份恢复功能支持对备份到 Azure Blob Storage 的数据设置 Azu
...
+
+> **警告：**
+>
+> 如果在快照备份执行过程中或者 PITR 日志备份过程中，才开启了对象锁定功能，快照备份或者日志备份可能会失败。需要重新启动快照备份或者是 PITR 日志备份任务来继续完成备份。
```
Comment (qiancai) [inline br/backup-and-restore-storages.md:223]:
```suggestion
> 如果在快照备份执行过程中或者 PITR 日志备份过程中，才开启了对象锁定功能，快照备份或者日志备份可能会失败。需要重新启动快照备份或者 PITR 日志备份任务来继续完成备份。
```

**Review by BornChanger — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/approve

**Pull request merged**